src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -18,20 +18,25 @@ function ExitThisPage ($module) {
   this.$indicatorContainer = null
   this.$overlay = null
   this.escCounter = 0
-  this.escTimerActive = false
   this.lastKeyWasModified = false
-  this.timeout = 5000 // milliseconds
+  this.timeoutTime = 5000 // milliseconds
+
+  // Store the timeout events so that we can clear them to avoid user keypresses overlapping
+  // setTimeout returns an id that we can use to clear it with clearTimeout,
+  // hence the 'Id' suffix
+  this.keypressTimeoutId = null
+  this.timeoutMessageId = null
 }
 
 /**
  * Create the <span> we use for screen reader announcements.
  */
 ExitThisPage.prototype.initUpdateSpan = function () {
   this.$updateSpan = document.createElement('span')
-  this.$updateSpan.setAttribute('aria-live', 'polite')
+  this.$updateSpan.setAttribute('role', 'status')
   this.$updateSpan.setAttribute('class', 'govuk-visually-hidden')
```

Comment:
My nomination for the Award for the Most Nitpicky of Nitpickings: setting the `className` property is slightly more direct than changing the attribute, for those precious ten-thousands of a second. 

```suggestion
  this.$updateSpan.className = 'govuk-visually-hidden'
```

(`className` also seems to be the more common usage in our current JS, though we don't seem to have much consistency in that regard.) 

I look forward to the ceremony. 🏆

Reply:
It'll be a white tie event 💅🏻

For real though I'm not sure why I didn't use `className` here. Makes much more sense. Good spot.